### PR TITLE
Allow removal of text message senders via UI

### PR DESCRIPTION
**Pull request by acodegirl**

### CODE REVIEW

File: api/migrations/versions/0244_add_sms_sender_archived_state.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+"""
+
+Revision ID: 0244
```
Comment (jonalport) [inline api/migrations/versions/0244_add_sms_sender_archived_state.py:3]: My PR #393 also adds a migration 244 😛

Comment (acodegirl) [reply api/migrations/versions/0244_add_sms_sender_archived_state.py:3]: @jonalport Thanks for letting me know. I will change it to 245.

**Comment by jonalport** [comment]
I will pull this down and look at it in more detail.

**Comment by jonalport** [comment]
https://govausites.atlassian.net/browse/NOT-14

**Comment by jonalport** [comment]
Rebased onto master and now seems to be working fine.

**Pull request merged**